src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,59 @@
+import { Link } from "react-router-dom";
+
+import './Footer.css';
+
+import FacebookIcon from '/Teal_MentorShip_Facebook.svg';
+import InstagramIcon from '/Teal_MentorShip_Instagram.svg';
+import LinkedInIcon from '/Teal_MentorShip_LinkedIn.svg';
+import SlackIcon from '/Teal_MentorShip_Slack.svg';
+import TwitterIcon from '/Teal_MentorShip_Twitter.svg';
+
+function Footer() {
+
+    return (
+        <section className="footer">
+            <div className="footer-text">
+                <p>&copy; 2023 MentorShip</p>
+            </div>
+            <footer>
+                <Link 
+                className="icon"
+                to="https://www.facebook.com/shecodesaustralia/" 
+                id="facebook"
+                >
+                    <img src={FacebookIcon} alt="" />
+                </Link>
+                <Link 
+                className="icon"
+                to="https://www.instagram.com/shecodesaus/?hl=en" 
+                id="instagram"
+                >
+                    <img src={InstagramIcon} alt="" />
+                </Link>
+                <Link 
+                className="icon"
+                to="https://www.linkedin.com/company/shecodesaustralia/" 
+                id="linkedin"
+                >
+                    <img src={LinkedInIcon} alt="" />
+                </Link>
+                <Link 
+                className="icon"
+                to="https://join.slack.com/t/shecodesaus/shared_invite/enQtNzQ3ODk1OTQzODc3LTQ2ZWFmNzM0N[…]JkMWMyYWVjMDU3ZThhODJmMTk4YmRjZGMzZDRhZDBhODQwNzdlYmJhNDg1YjE" 
```

Comment:
This link no longer works. Might be better to update with the one used on the She Codes website:
https://shecodesaus.slack.com/join/shared_invite/enQtNzQ3ODk1OTQzODc3LTQ2ZWFmNzM0NjE1MjA0NTI3YjJkMWMyYWVjMDU3ZThhODJmMTk4YmRjZGMzZDRhZDBhODQwNzdlYmJhNDg1YjE#/shared-invite/email

Reply:
Have just changed it